tests/src/common/lease.rs

```diff
@@ -245,6 +252,24 @@ fn do_swap<DownpaymentC, LeaseC>(
         exp_lease
     };
 
+    app.send_tokens(
+        Addr::unchecked(ica_addr),
+        Addr::unchecked(ADMIN),
+        &if currency::equal::<DownpaymentC, LeaseC>() {
+            vec![cwcoin(exp_borrow)]
+        } else {
+            vec![cwcoin(downpayment), cwcoin(exp_borrow)]
+        },
+    )
+    .unwrap();
+
+    app.send_tokens(
+        Addr::unchecked(ADMIN),
+        Addr::unchecked(ica_addr),
+        &[cwcoin(exp_swap_out)],
+    )
+    .unwrap();
+
     send_swap_response::<DownpaymentC, LeaseC>(app, lease_addr, exp_swap_out, remote_tx_count);
 
     check_state_opened(app, lease_addr);
```

Comment:
rename the too-generic naming of `fn send_response_and_expect` to `fn respond_and_expect_ibc_transfer`

Reply:
if you address a previous comment about requests and responses then the naming would be just `fn expect_ibc_transfer`. It would expect an IBC transfer request and send a response.